dist/vespa.spec

```diff
@@ -132,7 +133,6 @@ Requires: llvm-libs >= 8.0.0
 %define _extra_include_directory %{_vespa_deps_prefix}/include
 %endif
 Requires: java-11-openjdk
-Requires: openssl
```

Comment:
This should not be removed, but moved to inside %if 0%{?fedora}.
Similar for openssl-devel (buildReq).

Reply:
Ok. I will do that in another PR.